Update add track to playlist notification to handle previous format

sabrina-kiam opened the pull request:
### Description
we introduced a new format without handling previous formats that didn't have `playlist_owner_id`. this allows both types

### How Has This Been Tested?
testing on stage

_Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration._

isaacsolo reviewed (approved):
can the client handle playlist_owner_id None?

sabrina-kiam commented:
yup it was already handling playlist owner id is none before cuz that was the previous format, playlist owner id didn't exist. then i added it and changed it to expect it and this crashed bc older notifs don't have it 
client doesn't even parse for this notif as of now
